perftests/https/main.c

```diff
@@ -18,12 +18,12 @@ donereq(void * cookie, struct http_response * R)
 
 	/* Did we succeed? */
 	if (R == NULL) {
-		warn0("HTTP request failed");
+		warn0("HTTPS request failed");
 		return (-1);
 	}
 
 	/* Print status and headers. */
-	printf("HTTP status = %d\n", R->status);
+	printf("HTTPS status = %d\n", R->status);
```

Comment:
I'd say that it's still an HTTP status even if it's an HTTPS connection.

I agree with "HTTPS request failed" though since that could be for reasons including TLS issues.